python/sglang/srt/layers/attention/trtllm_mha_backend.py

```diff
@@ -411,6 +415,44 @@ def get_cuda_graph_seq_len_fill_value(self) -> int:
         """Get the fill value for sequence lengths in CUDA graph."""
         return 1
 
+    def _should_use_fused_fp8_path(
```

Comment:
So it's only enabled for Qwen? Is it possible to make it more general so that other models can use it?